### PR TITLE
add tutorial 1

### PR DESCRIPTION
**Pull request by njzjz**

### CODE REVIEW

File: source/_posts/tutorial1.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+Let's take a look at the training process of DeePMD-kit:
+
+{% mermaid graph LR %}
+A[Prepare data] --> B[Training]
+B --> C[Freeze the model]
+{% endmermaid %}
+
+What? Only three steps?
+Yes, it's that simple.
+Preparing data is converting the computational results of DFT to data that can be recongized by the DeePMD-kit.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:23]: Is this a typo: "recongized"
Should be: "recognized"

Comment (tuoping) [inline source/_posts/tutorial1.md:23]: Is this a typo: "recongized"
Should be: "recognized"

File: source/_posts/tutorial1.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import dpdata
+dpdata.LabeledSystem('OUTCAR').to('deepmd/npy', 'data', set_size=200)
+```
+
+In this example, we converted the computaional results of the VASP in the `OUTCAR` to the data format of the DeePMD-kit and saved in to a directory named `data`,
+where `npy` is the compressed format of the numpy, which is required by the DeePMD-kit training.
+We assume `OUTCAR` stores 1000 frames of molecular dynamics trajectory, then where will be 1000 points after converting.
+`set_size=200` means these 1000 points will be divided into 5 subsets, which is named as `data/set.000`~`data/set.004`, respectively.
+The size of each set is 200.
+In these 5 sets, `data/set.000`~`data/set.003` will be considered as the trainign set by the DeePMD-kit, and `data/set.004` will be considered as the test set.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:43]: A
```suggestion
In these 5 sets, `data/set.000`~`data/set.003` will be considered as the training set by the DeePMD-kit, and `data/set.004` will be considered as the test set.
```

File: source/_posts/tutorial1.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+The last set will be considered as the test set by the DeePMD-kit by default.
+If there is only one set, the set will be both the training set and the test set. (Of course, such the test set is meaningless.)
```
Comment (tuoping) [inline source/_posts/tutorial1.md:45]:
```suggestion
If there is only one set, the set will be both the training set and the test set. (Of course, such test set is meaningless.)
```

File: source/_posts/tutorial1.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+The strength of the DeePMD-kit is that the same training parameters are suitable for different systems, so we only need to slightly modify `input.json` to start training.
+Here is the first parameter to modify:
+
+```json
+"type_map":     ["O", "H"],
+```
+
+In the DeePMD-kit data, each atom type is numbered as an integer starting from 0.
+The parameter gices an element name to each atom in the numbering system.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:63]:
```suggestion
The parameter gives an element name to each atom in the numbering system.
```

File: source/_posts/tutorial1.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+```json
+"sel":       [46, 92],
+```
+
+Each number in this list gives the maximum number of atoms of each type among neighbor atoms of an atom.
+For example, `46` means there are at most 46 `O` (type `0`) neighbours.
+Here, our elements were modified to `A`, `B`, and `C`, so this parameters is also required to modify.
+What to do if you don’t know the maximum number of neighbors?
+You can be roughly estimate one by the density of the system, or try a number blindly.
+If it is not big enough, the DeePMD-kit will tell you.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:82]:
```suggestion
If it is not big enough, the DeePMD-kit will shoot WARNINGS. 
```

File: source/_posts/tutorial1.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+"systems":     ["../data/"],
+```
+
+to
+
+```
+"systems":     ["./data/"],
+```
+
+It is the reason that the directory to write to is `./data/` in the current directory.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:101]:
```suggestion
It is because that the directory to write to is `./data/` in the current directory.
```

File: source/_posts/tutorial1.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+Here I'd like to introduce the defination of the data system.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:102]:
```suggestion
Here I'd like to introduce the definition of the data system.
```

File: source/_posts/tutorial1.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+The DeePMD-kit considers that data with the same atomic number and elements can form a system.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:103]:
```suggestion
The DeePMD-kit considers that data with corresponding element types and atomic numbers form a system.
```

File: source/_posts/tutorial1.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+Our data is generated from a molecular dynamics simulation and meets this condition, so we can put them into one system.
+Dpdata also did so.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:105]:
```suggestion
Dpdata works the same way.
```

File: source/_posts/tutorial1.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+If data cannot be put into a system, multiple systems is required to be set as a list.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:106]:
```suggestion
If data cannot be put into a system, multiple systems is required to be set as a list here:
```json
 "training": {
        "systems": [system1, system2]
```
```

File: source/_posts/tutorial1.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+```
+
+Now we have succesfully set a input file! To start training, we execuate
+
+```sh
+dp train input.json
+```
+
+and wait for results. During the training process, we can see `lcurve.out` to observe the error reduction.
+Among them, Column 4 and 5 are the test and training errors of energy (normalize the number of atoms), and Column 6 and 7 are the test and training errors of the force.
```
Comment (tuoping) [inline source/_posts/tutorial1.md:129]:
```suggestion
Among them, Column 4 and 5 are the test and training errors of energy (normalized by the number of atoms), and Column 6 and 7 are the test and training errors of the force. 
```

**Review by tuoping — CHANGES_REQUESTED** [review]
